CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 3.0.0-nullsafety.2
+
+* Update Android library to version 2.8.6
+
```

Comment:
As I forgot to update the version in pubspec.yaml for PR #98, would someone please add these 2 lines too.
Migrate the example app to null safety
Fix a down cast issue

Reply:
I added your changes to the changelog in this PR.

Reply:
Thanks a lot!